contents/teams/sales-cs/objectives.mdx

```diff
@@ -1,9 +1,15 @@
 - **Objective**: Scale Sales the PostHog way
 - **Goals**
-  - Increase the amount of revenue from customers paying $20k+ by 30% (Cameron, Leon, Seb, Simon)
-  - Get good at measuring expansion and predicting churn in managed vs. non-managed accounts (Simon)
-  - We understand the best ways to expand managed accounts and create a playbook (Cameron)
-  - 10x the sales cycle from initial call to close - look at each step and see how we can improve (Leon, Seb)
-  - Create a fully automated process for self serve customers in Salesforce (Mine)
-  - We’ve closed HubSpot (Mine)
-  - Hire and onboard another AE (Simon)
\ No newline at end of file
+  - Hit individual quota targets (Cameron, Leon, Scott, Seb)
+  - Hit the team quota target (Simon)
+  - Move customers on legacy pricing to standard pricing (Cameron, Leon, Mine, Scott, Seb, Simon)
+  - Sell at least 1 paid Enterprise deal each (Cameron, Leon, Scott, Seb)
+  - Visit at least 1 customer in person (Cameron, Leon, Scott, Seb)
```

Comment:
@simfish85 not you for the Enterprise deal and visiting one customer objectives too?